### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by JanRei**
I have updated the German translation to fix some mistakes and to make it more consistent. There should be no untranslated messages anymore.

Also, I fixed some minor mistakes in English message file. The FAQ states that you cannot change your CryptPad password and you cannot share folders with others. This info seems to be outdated, so I changed it as well.

### CODE REVIEW

File: www/common/translations/messages.js [hunk 1]
```diff
@@ -1133,7 +1133,7 @@ define(function () {
     out.help.title = "Getting started";
     out.help.generic = {
         more: 'Learn more about how CryptPad can work for you by reading our <a href="/faq.html" target="_blank">FAQ</a>',
-        share: 'Use the share menu (<span class="fa fa-shhare-alt"></span>) to generate a link so collaborators can view or edit this pad',
+        share: 'Use the share menu (<span class="fa fa-share-alt"></span>) to generate a link so collaborators can view or edit this pad',
```
Comment (ansuz) [inline www/common/translations/messages.js]: This looks like a typo, but the double-h is deliberate. Various adblockers remove any element with classes including `fa-share`.

File: www/common/translations/messages.js [hunk 2]
```diff
@@ -442,7 +442,7 @@ define(function () {
     out.fm_restoreDialog = "Are you sure you want to restore {0} to its previous location?";
     out.fm_unknownFolderError = "The selected or last visited directory no longer exist. Opening the parent folder...";
     out.fm_contextMenuError = "Unable to open the context menu for that element. If the problem persist, try to reload the page.";
-    out.fm_selectError = "Unable to select the targetted element. If the problem persist, try to reload the page.";
+    out.fm_selectError = "Unable to select the targeted element. If the problem persist, try to reload the page.";
```
Comment (ansuz) [inline www/common/translations/messages.js]: Now that I'm looking at this I also notice that it should be `persists`. I can fix this after merging this PR, I'm just commenting here so I don't forget.

**Comment by ansuz** [comment]
@rhoio, @kpcyrd, @MacLemon... if you care to review the Deutsch-language changes before we merge, I'd appreciate it.

File: www/common/translations/messages.de.js [hunk 3]
```diff
@@ -753,44 +761,45 @@ define(function () {
...
+    out.home_host = "Dies ist eine unabhängige Installation der CrypPad-Software. Der Quellcode ist <a href=\"https://github.com/xwiki-labs/cryptpad\" target=\"_blank\" rel=\"noreferrer noopener\">auf GitHub</a> verfügbar.";
+    out.home_host_agpl = "CryptPad kann unter der Lizenz AGPL3 verbreitet werden";
+    out.home_ngi = "Gewinner beim NGI Award";
 
     //about.html
-    out.about_intro = 'CryptPad wurde erstellt im Research Team von <a href="http://xwiki.com">XWiki SAS</a>, einem kleinen Unternehmen in Paris, Frankreich, und Iasi, Rumänien. Das kernteam hat 3 Mitglieder, die an CryptPad arbeiten, sowie einige Mitwirkende innerhalb von XWiki SAS und außerhalb.';
-    out.about_core = 'Core Developers';
-    out.about_contributors = 'Key Contributors';
+    out.about_intro = 'CryptPad wurde erstellt im Forschungsteam von <a href="http://xwiki.com">XWiki SAS</a>, einem kleinen Unternehmen in Paris (Frankreich) und Iasi (Rumänien). Im Kernteam arbeiten 3 Mitglieder an CryptPad, außerdem gibt es einige Mitwirkende innerhalb und außerhalb von XWiki SAS.';
```
Comment (kpcyrd) [inline www/common/translations/messages.de.js]:
```suggestion
    out.about_intro = 'CryptPad wurde entwickelt im Forschungsteam von <a href="http://xwiki.com">XWiki SAS</a>, einem kleinen Unternehmen in Paris (Frankreich) und Iasi (Rumänien). Im Kernteam arbeiten 3 Mitglieder an CryptPad, außerdem gibt es einige Mitwirkende innerhalb und außerhalb von XWiki SAS.';
```

File: www/common/translations/messages.de.js [hunk 4]
```diff
@@ -803,170 +812,175 @@ define(function () {
...
+
+    out.features_f_reg = "Alle Funktionen für angemeldete Benutzer";
+    out.features_f_reg_note = "Du unterstützt die Entwicklung von CryptPad";
+    out.features_f_storage2 = "Mehr Speicherplatz";
+    out.features_f_storage2_note = "Zwischen 5 GB und 50 GB, abhängig vom gewählten Tarif";
+    out.features_f_support = "Schnellerer Support";
+    out.features_f_support_note = "Professioneller Support via E-Mail mit dem Team-Tarif";
     out.features_f_supporter = "Werde ein Unterstützer des Datenschutzes";
-    out.features_f_supporter_note = "Hilfe uns beweisen, dass Software mit eingebauten Datenschutz die Normalität sein sollten";
+    out.features_f_supporter_note = "Hilfe uns beim Beweis, dass Software mit eingebauten Datenschutz die Normalität sein sollte";
```
Comment (kpcyrd) [inline www/common/translations/messages.de.js]:
```suggestion
    out.features_f_supporter_note = "Hilfe uns zu beweisen dass Software mit eingebauten Datenschutz die Normalität sein sollte";
```

**Comment by kpcyrd** [comment]
@ansuz this patch removes a large amount of mistakes and improves the translation in many places.

Note that this patch also removes the inclusive gender asterisk from the translation.

File: www/common/translations/messages.de.js (hunk 4 above)
Comment (pajowu) [reply www/common/translations/messages.de.js]: Also: shouldn't that be "Hilf" or "Helfe" instead of "Hilfe"

Comment (rhoio) [reply www/common/translations/messages.de.js]: It should be "Hilf".

File: www/common/translations/messages.de.js [hunk 5]
```diff
@@ -24,62 +24,66 @@ define(function () {
...
+    out.onLogout = 'Du bist ausgeloggt. {0}Klicke hier{1}, um dich wieder einzuloggen,<br>oder drücke <em>Escape</em>, um dein Pad schreibgeschützt zu benutzen.';
+    out.wrongApp = "Der Inhalt dieser Echtzeitsitzung kann nicht in deinem Browser angezeigt werden. Bitte lade die Seite neu.";
+    out.padNotPinned = 'Dieses Pad wird nach 3 Monaten ohne Aktivität auslaufen, {0}logge dich ein{1} oder {2}registriere dich{3}, um das Auslaufen zu verhindern.';
+    out.anonymousStoreDisabled = "Der Webmaster dieses CryptPad-Servers hat die anonyme Verwendung des Speichers deaktiviert. Du musst dich einloggen, um CryptDrive zu verwenden.";
+    out.expiredError = 'Dieses Pad ist abgelaufen und ist nicht mehr verfügbar.';
+    out.deletedError = 'Dieses Pad wurde von seinem Besitzer gelöscht und ist nicht mehr verfügbar.';
+    out.inactiveError = 'Dieses Pad ist wegen Inaktivität gelöscht worden. Drücke Esc, um ein neues Pad zu erstellen.';
+    out.chainpadError = 'Ein kritischer Fehler ist beim Aktualisieren deines Inhalts aufgetreten. Diese Seite ist schreibgeschützt, damit du sicherstellen kannst, dass kein Inhalt verloren geht.<br>'+
```
[the remark this answers is not in the document]
Comment (rhoio) [reply www/common/translations/messages.de.js]: They may be optional (I don't know) but they help readers parse long sentences and improve readability. I'd recommend against removing them.

File: www/common/translations/messages.de.js [hunk 6]
```diff
@@ -753,44 +761,45 @@ define(function () {
     out.mdToolbar_code = "Code";
 
     // index.html
-    out.home_product = "CryptPad ist eine alternative zu verbreiteten Office- und Clouddienste mit eingebauten Datenschutz. Mit CryptPad, der gesamten Inhalt ist verschlüsselt, bevor es geschickt wird. Das heisst, dass keiner hat Zugang zum Inhalt, ausser du gibst den Schlüssel aus. Selbst die Softwarehersteller haben diesen Zugang nicht.";
-    out.home_host = "Dieses CryptPad Server ist eine unabhängige Installation des Communitysoftwares. Das Quellcode ist <a href=\"https://github.com/xwiki-labs/cryptpad\" target=\"_blank\" rel=\"noreferrer noopener\">auf GitHub</a> verfügbar.";
-    out.home_host_agpl = "CryptPad kann durch die Lizenz AGPL3 verbreitet werden";
-    out.home_ngi = "Gewinner beim NGI Award";
 
+    out.home_product = "CryptPad ist eine Alternative mit eingebauten Datenschutz zu verbreiteten Office- und Clouddiensten. Mit CryptPad wird der gesamte Inhalt verschlüsselt, bevor er an den Server gesendet wird. Das bedeutet, dass keiner auf den Inhalt zugreifen kann, es sei denn du gibst die Schlüssel weiter. Selbst wir haben diesen Zugriff nicht.";
```
Comment (rhoio) [inline www/common/translations/messages.de.js]: It should be "mit eingebautem Datenschutz"

**Comment by ansuz** [comment]
I just realized that this PR is against the _master_ branch, and that we've recently received a PR against _staging_, so there might be some conflicts.

I'm going to edit this PR to target the staging branch so we can see if there are any conflicts.

**Comment by ansuz** [comment]
@promasu, you'll probably want to review these changes.

File: www/common/translations/messages.de.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (promasu) [reply www/common/translations/messages.de.js]: These are not optional. Please let them stay there :D

File: www/common/translations/messages.de.js (hunk 4 above)
Comment (promasu) [reply www/common/translations/messages.de.js]: Should be "Hilf uns zu beweisen, dass [...]"

File: www/common/translations/messages.de.js [hunk 7]
```diff
@@ -24,62 +24,66 @@ define(function () {
...
+    out.wrongApp = "Der Inhalt dieser Echtzeitsitzung kann nicht in deinem Browser angezeigt werden. Bitte lade die Seite neu.";
+    out.padNotPinned = 'Dieses Pad wird nach 3 Monaten ohne Aktivität auslaufen, {0}logge dich ein{1} oder {2}registriere dich{3}, um das Auslaufen zu verhindern.';
+    out.anonymousStoreDisabled = "Der Webmaster dieses CryptPad-Servers hat die anonyme Verwendung des Speichers deaktiviert. Du musst dich einloggen, um CryptDrive zu verwenden.";
+    out.expiredError = 'Dieses Pad ist abgelaufen und ist nicht mehr verfügbar.';
+    out.deletedError = 'Dieses Pad wurde von seinem Besitzer gelöscht und ist nicht mehr verfügbar.';
+    out.inactiveError = 'Dieses Pad ist wegen Inaktivität gelöscht worden. Drücke Esc, um ein neues Pad zu erstellen.';
+    out.chainpadError = 'Ein kritischer Fehler ist beim Aktualisieren deines Inhalts aufgetreten. Diese Seite ist schreibgeschützt, damit du sicherstellen kannst, dass kein Inhalt verloren geht.<br>'+
+                        'Drücke <em>Esc</em>, um das Pad schreibgeschützt zu lesen, oder lade es neu, um die Bearbeitung fortzusetzen.';
```
[the remark this answers is not in the document]
Comment (promasu) [reply www/common/translations/messages.de.js]: Remove the first comma and let the second one stay there. Then it's fine

File: www/common/translations/messages.js [hunk 8]
```diff
@@ -1041,7 +1041,7 @@ define(function () {
         },
         change: {
             q: "What if I want to change my password?",
-            a: "It is not currently possible to change your CryptPad password, though we are planning to develop this functionality very soon."
+            a: "You can change your CryptPad password in settings."
```
Comment (promasu) [inline www/common/translations/messages.js]: It should be "in the settings"

Comment (promasu) [inline www/common/translations/messages.js]: It should be "in the settings"

**Comment by ansuz** [comment]
I've merged this PR manually into our _soon_ branch, which will be merged to master for today's release. As I mentioned in our chat channel, we're moving towards a JSON format for our translation files to allow us to use [Weblate](https://weblate.org). I've run a migration to transform `messages.de.js` into `message.de.json`.

You can review the translation in its current state here: https://github.com/xwiki-labs/cryptpad/blob/soon/www/common/translations/messages.de.json

After today's release we'll merge all of this back up to our staging branch where we normally do our development. Until we manage to launch our weblate instance, we'll prefer to take PRs against the _staging_ branch to avoid conflicts, and to incorporate any keys that we introduce in between releases.

**Pull request merged**